### ISSUE TITLE
Cant load my save states on ARM64

### ISSUE THREAD

**Issue by Askmewho** [SourMesen/Mesen#767]
It doesnt happens all the time, just sometimes... this is the output I have, would you like a gdb bt??
I will retry it on RPI4...maybe its a gpu driver problem
```
pi@rockpi:~/Desktop/mesen/x64/Release$ mono Mesen.exe
Gtk-Message: 02:34:12.916: Failed to load module "gail"

** (mono:7822): WARNING **: 02:34:12.928: (../atk-adaptor/bridge.c:993):atk_bridge_adaptor_init: runtime check failed: (root)
Could not parse file "/home/pi/.local/share/applications/xfce4-about.desktop": Key file does not have group ?Desktop Entry?
Stacktrace:

/proc/self/maps:
aaaae476c000-aaaae4b78000 r-xp 00000000 b3:01 30019                      /usr/bin/mono-sgen
aaaae4b88000-aaaae4b8f000 r--p 0040c000 b3:01 30019                      /usr/bin/mono-sgen
aaaae4b8f000-aaaae4b94000 rw-p 00413000 b3:01 30019                      /usr/bin/mono-sgen
aaaae4b94000-aaaae4c25000 rw-p 00000000 00:00 0 
aaab193ea000-aaab3efeb000 rw-p 00000000 00:00 0                          [heap]
ffff40000000-ffff40021000 rw-p 00000000 00:00 0 
ffff40021000-ffff44000000 ---p 00000000 00:00 0 
ffff48000000-ffff489e5000 rw-p 00000000 00:00 0 
ffff489e5000-ffff4c000000 ---p 00000000 00:00 0 
ffff4c000000-ffff4c1e4000 rw-p 00000000 00:00 0 
ffff4c1e4000-ffff50000000 ---p 00000000 00:00 0 
ffff50000000-ffff50021000 rw-p 00000000 00:00 0 
ffff50021000-ffff54000000 ---p 00000000 00:00 0 
ffff54669000-ffff5483c000 rw-s 00000000 00:06 12875                      /dev/dri/renderD128
ffff5483c000-ffff54a00000 rw-s 00000000 00:06 12875                      /dev/dri/renderD128
ffff54a00000-ffff54b00000 rw-s 00000000 00:06 12875                      /dev/dri/renderD128
ffff54b00000-ffff54c00000 rw-p 00000000 00:00 0 
ffff54cf1000-ffff54eda000 rw-s 00000000 00:06 12875                      /dev/dri/renderD128
ffff54eda000-ffff5509e000 rw-s 00000000 00:06 12875                      /dev/dri/renderD128
ffff550a4000-ffff55277000 rw-s 00000000 00:06 12875                      /dev/dri/renderD128
ffff55277000-ffff5543b000 rw-s 00000000 00:06 12875                      /dev/dri/renderD128
ffff5543b000-ffff5560e000 rw-s 00000000 00:06 12875                      /dev/dri/renderD128
ffff55700000-ffff55800000 rw-p 00000000 00:00 0 
ffff55862000-ffff559f9000 rw-s 00000000 00:06 12875                      /dev/dri/renderD128
ffff55a25000-ffff55b25000 rw-s 00000000 00:06 12875                      /dev/dri/renderD128
Memory around native instruction pointer (0xffff83d86d88):
0xffff83d86d78  96 1e 00 f9 88 06 01 39 8a 02 40 f9 e9 17 40 f9  .......9..@...@.
0xffff83d86d88  48 a5 41 b9 29 01 17 8b e9 53 00 f9 48 06 00 34  H.A.)....S..H..4
0xffff83d86d98  ed 33 49 a9 0b 0a 80 52 89 16 40 f9 1f 05 00 71  .3I....R..@....q
0xffff83d86da8  ab 31 0b 9b 6b 89 40 39 29 79 6b b8 81 00 00 54  .1..k.@9)yk....T

Native stacktrace:

	mono(+0xe5d54) [0xaaaae4851d54]
	mono(+0xe601c) [0xaaaae485201c]
	mono(+0xa83e0) [0xaaaae48143e0]
	mono(+0x34af4) [0xaaaae47a0af4]
	linux-vdso.so.1(__kernel_rt_sigreturn+0) [0xffffa16df698]
	/home/pi/Desktop/mesen/x64/Release/libMesenCore.dll(_ZN9HdNesPack7ProcessEP12HdScreenInfoPjR18OverscanDimensions+0x3a8) [0xffff83d86d88]
	/home/pi/Desktop/mesen/x64/Release/libMesenCore.dll(_ZN13HdVideoFilter11ApplyFilterEPt+0x34) [0xffff83d8b65c]
	/home/pi/Desktop/mesen/x64/Release/libMesenCore.dll(_ZN15BaseVideoFilter9SendFrameEPtj+0x33c) [0xffff83ee65d0]
	/home/pi/Desktop/mesen/x64/Release/libMesenCore.dll(_ZN12VideoDecoder11DecodeFrameEb+0x48) [0xffff83dd12dc]
	/home/pi/Desktop/mesen/x64/Release/libMesenCore.dll(_ZN12VideoDecoder12DecodeThreadEv+0x54) [0xffff83dd1cf4]
	/usr/lib/aarch64-linux-gnu/libstdc++.so.6(+0xcb804) [0xffff8f376804]
	/lib/aarch64-linux-gnu/libpthread.so.0(+0x896c) [0xffffa159796c]
	/lib/aarch64-linux-gnu/libc.so.6(+0xd3ecc) [0xffffa14f0ecc]
Pkilling 0xffff9d7431a0 from 0xffff575371a0
Pkilling 0xffff809ff1a0 from 0xffff575371a0
Pkilling 0xffff57fff1a0 from 0xffff575371a0
Pkilling 0xffffa16da720 from 0xffff575371a0
Pkilling 0xffff577381a0 from 0xffff575371a0
Pkilling 0xffff802291a0 from 0xffff575371a0
Pkilling 0xffff9d9441a0 from 0xffff575371a0
Pkilling 0xffff9d5421a0 from 0xffff575371a0
Pkilling 0xffff807fe1a0 from 0xffff575371a0
Pkilling 0xffffa12c31a0 from 0xffff575371a0
Pkilling 0xffff785771a0 from 0xffff575371a0
Pkilling 0xffff8042a1a0 from 0xffff575371a0
Entering thread summarizer pause from 0xffff575371a0
Finished thread summarizer pause from 0xffff575371a0.
domain required for stack walk

Waiting for dumping threads to resume


Debug info from gdb:

mono_gdb_render_native_backtraces not supported on this platform, unable to find gdb or lldb

=================================================================
Got a SIGSEGV while executing native code. This usually indicates
a fatal error in the mono runtime or one of the native libraries 
used by your application.
=================================================================

Aborted
```

**Comment by SourMesen** [comment]
The call stack in the crash implies it may have something to do with HD packs (assuming you always get a similar crash)
Can you reproduce the crash if you disable HD packs?

**Comment by Askmewho** [comment]
Yeah, the problem doesn't happen if you are not utilizing hd packa

**Comment by SourMesen** [comment]
Thanks for the confirmation - this should be fixed by f2112dbadca463fc384942c2050e1f09fa9ad3a3
